### DISCUSSION TITLE
General front-end implementation

### DISCUSSION THREAD

**Discussion by MaximVanhove** [spatie/laravel-query-builder#504] [Show and tell] [answered]
Hi!

I worked for months on a javascript library to work great with the query builder. It is inspired by laravel models and laravel routes. It's not framework specific, but it works great with Vue. I was hoping that you could add it to this page: [https://docs.spatie.be/laravel-query-builder/v2/advanced-usage/front-end-implementation/](https://docs.spatie.be/laravel-query-builder/v2/advanced-usage/front-end-implementation/)

Check it out: [https://www.npmjs.com/package/elodo](https://www.npmjs.com/package/elodo)

We use it in all our SPA projects at [Starring Jane](https://www.starringjane.com/)

P.S.: Thank you for creating this awesome package 👍

**Comment by freekmurze** [comment, marked as answer]
Feel free to send a PR to that page. There's an edit button that allows you to create a PR quickly.
